<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="appending_t">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>181</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Inserting_t">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>211</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>appending</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Inserting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="append_b">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>append</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insert_html_b">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>160</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert_html</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="insert_b">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>177</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>insert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
